ee/migrations/0009_null_definition_descriptions.py

```diff
@@ -6,7 +6,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ("ee", "0007_dashboard_permissions"),
+        ("ee", "0008_deprecated_old_tags"),
```

Comment:
What's going on here? I don't think we should be touching migrations that already ran in prod